src/collapse/mod.rs

```diff
@@ -17,9 +17,24 @@ pub mod perf;
 
 use std::fs::File;
 use std::io::{self, Write};
+use std::mem;
 use std::path::Path;
+use std::sync::Arc;
 
-const READER_CAPACITY: usize = 128 * 1024;
+use chashmap::CHashMap;
+use fnv::FnvHashMap;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    #[doc(hidden)]
```

Comment:
It'd be good to leave a comment as to why this has to be `pub`.

Reply:
Makes sense. As you've probably already surmised, it's because we use it in `main.rs`.